### PR TITLE
New metadata features

### PR DESCRIPTION
**Pull request by Derstilon**
Add metadata to simulation results
Keep input files even when simulation suceeded

### CODE REVIEW

File: yaptide/celery/tasks.py [hunk 1]
```diff
@@ -66,7 +67,18 @@ def run_simulation(self, param_dict: dict, raw_input_dict: dict):
 
         result: dict = pymchelper_output_to_json(estimators_dict)
 
-        return {'result': result, 'input': raw_input_dict, 'end_time': datetime.utcnow(), 'cores': runner_obj.jobs}
+        return {
+            'result': result, 
+            'metadata': {
+                'source': 'YAPTIIDE' if 'metadata' in raw_input_dict else 'Input files',
```
Comment (grzanka) [inline yaptide/celery/tasks.py]:
```suggestion
                'source': 'YAPTIDE' if 'metadata' in raw_input_dict else 'Input files',
```

**Comment by grzanka** [comment]
@Derstilon fix failing CI tests

File: yaptide/celery/tasks.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-from yaptide.celery.worker import celery_app
+'from yaptide.celery.worker import celery_app
```
Comment (p1003) [inline yaptide/celery/tasks.py]:
```suggestion
from yaptide.celery.worker import celery_app
```

File: yaptide/celery/tasks.py [hunk 3]
```diff
@@ -269,3 +286,4 @@ def cancel_simulation(task_id: str) -> bool:
     # Currently this task does nothing because to working properly it requires changes in pymchelper
     print(task_id)
     return False
+'
```
Comment (p1003) [inline yaptide/celery/tasks.py:289]:
```suggestion

```

Comment (p1003) [inline yaptide/celery/tasks.py]:
```suggestion
    return False

```

File: yaptide/celery/tasks.py [hunk 4]
```diff
@@ -17,11 +17,19 @@
 from ..converter.converter.api import get_parser_from_str, run_parser  # skipcq: FLK-E402
 
 
-def save_input_files(input_files: dict, output_dir: str):
-    """Function used to save input files"""
-    for key, value in input_files.items():
+def write_input_files(param_dict: dict, raw_input_dict: dict, output_dir: str):
+    """
+    Function used to write input files to output directory.
```
Comment (p1003) [inline yaptide/celery/tasks.py]:
```suggestion
    Function used to write input files to output directory.
```

**Review by p1003 — APPROVED** [review]

**Pull request merged**